Src/Support/GoogleApis.Core/Apis/Requests/Parameters/ParameterUtils.cs

```diff
@@ -29,6 +30,9 @@ namespace Google.Apis.Requests.Parameters
     /// </summary>
     public static class ParameterUtils
     {
+        //TODO: Find out how to resolve the issue with setting up the Logger:
+        private static readonly ILogger Logger = ApplicationContext.Logger.ForType<ParameterUtils>();

```

Comment:
What happens if you change it to `ApplicationContext.Logger.ForType(typeof(ParameterUtils))`
IMO, it should work